tools/bmh_test/create_bmh.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# -------------------------------------------------------------------------------------------
+# Description:    This script creates a virtual machine using virt-install,
+#                 adds the virtual machine to VBMC (Virtual BMC) for out-of-band management,
+#                 and applies the configuration to Kubernetes
+#
+# Usage:          make tilt-up -> press button in the right upper corner to create bmhs
+#                 /tools/bmh_test/create_bmh.sh
+# 
+# Prerequecites:  a network with ip address of 192.168.222.1 named bmo-test-network and
+#                 VBMC runing
+# -------------------------------------------------------------------------------------------
+
+set -euxo pipefail
+
+# Set default values
+NAME="${1:?}"
+VBMC_PORT="${2:?}"
+CONSUMER="${3}"
```

Comment:
Btw, now that you have `set -u`, this will fail unless `$3` and `$4` are provided.

You want to have
```suggestion
CONSUMER="${3:-}"
```
So at least empty value is substituted, which works nicely with the `[[ -n ... ]]` which we have later, since it is checking undefined or empty value.